docs/source/format/driver_manifests.rst

```diff
@@ -313,14 +318,15 @@ to control which directories will be searched for manifests, with the behavior b
     .. tab-item:: R
        :sync: r
 
-       Use ``adbc_driver(... , load_flags = adbc_load_flags())`` to pass options to the driver manager
+       Use ``adbc_driver(..., load_flags = adbc_load_flags())`` to pass options to the driver manager
        regarding how to locate drivers specified by manifest.
```

Comment:
We should either add a single space to each line here so it's in a quote box like the other tabs, or we should remove the single leading space from each line so it doesn't use a quote box.

Basically, all of the tabs should either line up with the `:` of `:sync:` or the `s`. We should be consistent

Reply:
Ahh, I see it now. Good eye.

Fixed in 13b741d2893e19994c0ccb24d284b38b6198ed08.

Reply:
(They all render as not-quote-boxes, just normally now)